### ISSUE TITLE
update nest imports to be compatible w/ tf2

### ISSUE THREAD

**Issue by guo-max** [tensorflow/privacy#29]
I tried to modify "tensorflow/python/keras/engine/training_utils.py" according the keras_tutorial:def get_loss_function():
    ...
-   return losses.LossFunctionWrapper(loss_fn, name=loss_fn.__name__)
+   return losses.LossFunctionWrapper(loss_fn,
+                                     name=loss_fn.__name__,
+                                     reduction=losses_impl.Reduction.NONE)

I found out only tensorflow>1.12 has the function "get_loss_function(loss):". 
And only tensorflow2.0 has the line of code"return losses.LossFunctionWrapper(loss_fn, name=loss_fn.__name__)". So I guess the tutorial use tensorflow2.0.0?

But When import privacy.analysis. privacy_ledger.py, there are lots of error. like "nest = tf.contrib.framework.nest" since tf2.0 does not have module"contrib".

My question is which tensorflow version should I use if I try to use keras. 
Thanks

**Comment by npapernot** [comment]
That's correct you should use 2.0, I'm rolling out a fix because `nest` has moved from `tf.contrib` to `tf.nest` in 2.0. I'll close the issue when the fix has rolled out.

**Comment by AnkitSiva** [comment]
Looks like this fix need to be applied to `privacy.dp_query.normalized_query.py` as well. Line 26 still has `nest = tf.contrib.framework.nest` instead of the block:
```python
if LooseVersion(tf.__version__) < LooseVersion('2.0.0'):
  nest = tf.contrib.framework.nest
else:
  nest = tf.nest
```

**Comment by npapernot** [comment]
This will be fixed in an update being rolled out
